.github/workflows/update-test-results.yml

```
@@ -15,6 +15,7 @@ name: Update test results
 permissions:
     contents: write
     actions: write
+    workflows: write
```

actionlint (1.7.7)

unknown permission scope "workflows". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)